types/v1/metrics.go

```diff
@@ -24,6 +26,29 @@ type VolumeMetrics struct {
 	Name              string  `json:"Name"`
 }
 
+// VolumeStats is used to store the collected metrics
+// TODO: Make this generic, so that it can be used by mayactl
+// and other components of maya.
+type VolumeStats struct {
+	Iqn                 string      `json:"iqn"`
+	Reads               json.Number `json:"ReadIOPS"`
```

Comment:
json tag should also be changed to Reads instead of ReadIOPS.

Reply:
This demands the changes in Jiva, mayactl and istgt as well and hence things may break. So it will be better if we can create a separate task for this and address this in new PR.